docs/how-to-build-with-ddn/with-sqlserver.mdx

```diff
@@ -0,0 +1,462 @@
+
+### Step 4. Start the local SQL Server container
+
+```sh title="Start a SQL Server instance"
+docker pull mcr.microsoft.com/mssql/server:2019-latest
+docker run -e "ACCEPT_EULA=Y" -e "MSSQL_SA_PASSWORD=Password123" -p 1433:1433 --name sqlserver -d mcr.microsoft.com/mssql/server:2019-latest
+```
+
+Download [Visual Studio Code](https://code.visualstudio.com/) and install the
+[SQL Server extension](https://marketplace.visualstudio.com/items?itemName=ms-mssql.mssql) in it.
+
+```plaintext title="Connect to the database using the connection string:"
+Server=localhost,1433;Database=master;Uid=sa;Password=Password123;TrustServerCertificate=true
+```
+
+```SQL title="Create a table in the database:"
+CREATE TABLE users (user_id int, name varchar(255), age int);
+```
+
+```SQL title="Then, seed the table:"
+INSERT INTO users (user_id, name, age) VALUES (1, 'Alice', 25), (2, 'Bob', 30), (3, 'Charlie', 35);
+```
+
+```SQL title="You can verify this by running:"
+SELECT * FROM users;
+```
+
+You should see a list of users returned.
+
+:::tip
+
+Alternativey, other applications and tools can also be used to seed the database, such as
+
+1. [Microsoft Azure Studio](https://azure.microsoft.com/en-us/products/data-studio)
+2. `mssql-tools` shell to start the SQL prompt:
+
+   a. Install
+   [mssql-tools](https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-setup-tools?view=sql-server-ver16&tabs=redhat-install)
+
+   b. Run the following in a bash shell:
+
+   ```sh
+   docker exec -it sqlserver /opt/mssql-tools18/bin/sqlcmd -S localhost -U SA -P 'Password123' -d 'master' -C
+   ```
+
+   :::note
+
+   1. The flag `-C` is used to apply the `trust server certificate` set to true. This should only be used for local
+      database and not on production databases.
+   2. Each SQL statement in the mssql-tools interactive shell needs a `GO` statement at the end for the SQL statement to
+      process. For example,
+      ```
+      SELECT * FROM users;
+      GO
+      ```
+
+   :::
+
+:::
+
+### Step 5. Introspect your SQL Server database
+
+```sh title="Next, use the CLI to introspect your SQL Server database:"
+ddn connector introspect my_sqlserver
+```
+
+After running this, you should see a representation of your database's schema in the
+`app/connector/my_sqlserver/configuration.json` file; you can view this using `cat` or open the file in your editor.
+
+```sh title="Additionally, you can check which resources are available — and their status — at any point using the CLI:"
+ddn connector show-resources my_sqlserver
+```
+
+### Step 6. Add your model
+
+```sh title="Now, track the table from your SQL Server database as a model in your DDN metadata:"
+ddn models add my_sqlserver users
+```
+
+Open the `app/metadata` directory and you'll find a newly-generated file: `Users.hml`. The DDN CLI will use this Hasura
+Metadata Language file to represent the `users` table from SQL Server in your API as a
+[model](/reference/metadata-reference/models.mdx).
+
```

Comment:
Let's also add a comment on how users can track all the available models using `ddn models add *`